illumos-utils/src/opte/illumos/port_manager.rs

```diff
@@ -424,6 +427,36 @@ impl PortManager {
         }
         Ok(())
     }
+
+    pub fn set_virtual_nic_host(
+        &self,
+        _interface_id: Uuid,
```

Comment:
`interface_id` is required for the simulated sled agent, and sticking it in the real sled agent too makes the http entrypoints the same - Nexus can PUT or DELETE the same payload to either one. It has to remain in the HTTP endpoint of the real sled agent, however, otherwise we'll see

```
thread 'main' panicked at 'failed to register entrypoints: path parameters are not consumed (interface_id)', sled-agent/src/http_entrypoints.rs:49:9
```

when generating the openapi JSON. Removed `interface_id` propagation for the real sled agent in 7407c31b